src/Workspaces/Core/Portable/Workspace/ProjectSystem/ProjectSystemProject.cs

```diff
@@ -653,12 +654,21 @@ await _projectSystemProjectFactory.ApplyBatchChangeToWorkspaceMaybeAsync(useAsyn
                         newSolution: solutionChanges.Solution.RemoveProjectReference(Id, projectReference));
                 }
 
+                // Analyzer reference removing...
+                {
```

Comment:
>{ [](http://example.com/codeflow?start=16&length=1)

nit: might be nice to add a Count > 0 check for both these scopes, just to make it very evident that we're not doing anything in those cases.

Reply:
sure.